projects/step-frontend/src/lib/modules/timeseries/chart/time-series-chart.component.ts

```diff
@@ -26,7 +26,7 @@ import uPlot = require('uplot');
   styleUrls: ['./time-series-chart.component.scss'],
 })
 export class TimeSeriesChartComponent implements OnInit, AfterViewInit, OnChanges {
-  private readonly HEADER_WITH_FOOTER_SIZE = 80;
+  private readonly HEADER_WITH_FOOTER_SIZE = 48;
   readonly WRAPPER_PADDING = '12px';
```

Comment:
This was not newly introduced but having a style hardcoded here seems weird. Why is this not something defined by the styles?

Reply:
Ah it seems that you removed it in the template, so we can remove it here as well now?

Reply:
yes, we can remove it. I will take care of it